schema/swagger.yml

```diff
@@ -16,7 +16,7 @@ swagger: '2.0'
 info:
   description: Lisk API Documentation
   title: Lisk API Documentation
-  version: '1.0.25'
+  version: '1.0.26'
```

Comment:
@diego-G Just to highlight there is another open PR pointing to exact same change. 

https://github.com/LiskHQ/lisk/pull/1940/files#diff-053c90a95ed06515a37c330053ee1a63R19

So when that will be merged you will have conflicts. I suggest to merge both changes under `1.0.26` and publish it to swagger hub. 

Everything else LGTM.